src/modules/meetings/adapter/zoom.adapter.ts

```diff
@@ -0,0 +1,118 @@
+            }
+            const resp = await axios.post(this.zoom_url, createMeetingDto, {
+                headers: {
+                    Authorization: `Bearer ${token}`,
+                    "Content-Type": "application/json",
+                },
+            });
+            const { id, password } = resp.data;
+            return { id, password }
+            // return resp.data;
+        }
+        catch (err) {
+            if (err.response && err.response.status === 400) {
+                throw new Error('Bad request: ' + err.response.data.message);
+            } else if (err.response && err.response.status) {
+                throw new Error(`Error ${err.response.status}: ${err.response.data.message}`);
+            } else {
+                throw new Error('Internal server error');
+            }
+        }
+    }
+    async getMeetingList() {
+        try {
+            const token = await this.getToken();
+            if (!token) {
+                throw new Error('Failed to retrieve access token');
+            }
+            const resp = await axios.get(this.zoom_url, {
+                headers: {
+                    Authorization: `Bearer ${token}`,
+                    "Content-Type": "application/json",
+                },
+            });
+
+            const meetings = resp.data.meetings;
+            const result = meetings.map((obj) =>
+                ["id", "topic"].reduce((newObj, key) => {
+                    newObj[key] = obj[key];
+                    return newObj;
+                }, {})
+            );
+            return result;
+
+        } catch (e) {
+            throw new Error(e.message || 'Internal server error');
+        }
+    }
+
+    async getToken() {
+        try {
+            const base64Credentials = Buffer.from(`${this.client_id}:${this.secret_token}`).toString('base64');
+            const resp = await axios.post(
+                `${this.zoom_auth_url}?grant_type=account_credentials&account_id=${this.account_id}`,
+                {},
+                {
+                    headers: {
+                        Authorization: `Basic ${base64Credentials}`,
+                    },
+                }
+            );
+            return resp.data.access_token;
+        } catch (e) {
+            return e.message;
+        }
+    }
+
+    async updateMeeting(meetingId, updateeMeetingDto: UpdateeMeetingDto) {
+        try {
+            const token = await this.getToken();
+            if (!token) {
+                throw new Error('Failed to retrieve access token');
+            }
+            const resp = await axios.patch(`${this.zoom_update_get_url}/${meetingId}`, updateeMeetingDto, {
+                headers: {
+                    Authorization: `Bearer ${token}`,
+                    "Content-Type": "application/json",
+                },
+            });
+            return resp;
+        }
+        catch (e) {
+            return e;
+        }
+    }
+    checkZoomConfig(): string | null {
```

Comment:
@poojakarma @vaivk369 Check is this method needed for all provide and implement from interface.